addons/account/models/account_move.py

```diff
@@ -1998,6 +2001,13 @@ def _compute_next_payment_date(self):
         for move in self:
             move.next_payment_date = min([line.payment_date for line in move.line_ids.filtered(lambda l: l.payment_date and not l.reconciled)], default=False)
 
+    @api.depends('line_ids.tax_ids', 'fiscal_position_id')
+    def _compute_incorrect_taxes_warning(self):
+        for move in self:
+            move.display_incorrect_taxes_warning = (
+                True if move.fiscal_position_id and not move.invoice_line_ids._validate_tax_alignment_with_fpos() else False
```

Comment:
```suggestion
                move.fiscal_position_id and not move.invoice_line_ids._validate_tax_alignment_with_fpos()
```
nitpick and should be only triggered when move is posted (if I understood the spec clearly but still verify it PO)

Reply:
As discuss with PO, He need the warning even if move is in Draft (Same as `Update Taxes and Accounts` button).